kitchensink/index.html

```diff
@@ -148,8 +148,86 @@
             </div>
 
           </div>
+
+          <div id="app-header-menu">
+            <div class="nav navbar-nav app-header-menu-right">
+              <ul class="nav nav-tabs nav-tabs-light" role="tablist">
+                <li class="dropdown" role="presentation">
+                  <a class="pull-left" href="" role="tab">App Header Menu</a>
+                  <a class="pull-right" href="" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    <span class="caret"></span>
+                    <span class="sr-only">Toggle Dropdown</span>
+                  </a>
+
+                  <ul class="dropdown-menu" id="app-header-dropdown-menu" role="menu" aria-hidden="true" aria-label="submenu">
+
+                    <li class="active" role="presentation">
+                      <a href="http://www.google.com" target="_blank" role="tab">
+                        Active App Link
+                      </a>
+                    </li>
+
+                    <li class="divider"></li>
+                    <li role="presentation">
+                      <a href="">Normal App Link</a>
+                    </li>
+
+                    <li class="divider"></li>
+                    <li class="disabled" role="presentation">
+                      <a role="tab">DIsabled App Link</a>
+                    </li>
+
+                    <li class="divider"></li>
+                    <li class="dropdown" role="presentation">
```

Comment:
This dropdown markup is duplicated elsewhere in index.html.

Reply:
Since kitchen sink is one large html file and is currently not using any templating mechanism, I did not see it appropriate to use templating for this isolated instance, unless we are planning to refactor the entire kitchen sink html code in the future